Example/Tests/DiscoveryTaskTests.swift

```diff
@@ -0,0 +1,97 @@
+import XCTest
+@testable import OktaAuth
+
+class DiscoveryTaskTests: XCTestCase {
+
+    var apiMock: OktaApiMock!
+    
+    override func setUp() {
+        super.setUp()
+        apiMock = OktaApiMock()
+    }
+
+    override func tearDown() {
+        apiMock = nil
+        super.tearDown()
+    }
+    
+    func testRunSucceeded() {
+        apiMock.configure(response: self.validOIDConfigDictionary)
+        
+        runAndWaitDiscovery(config: validConfig) { oidConfig, error in
+            XCTAssertNil(error)
+            XCTAssertNotNil(oidConfig)
+        }
+    }
+    
+    func testRunApiError() {
+        apiMock.configure(error: OktaError.APIError("Test Error"))
+        
+        runAndWaitDiscovery(config: validConfig) { oidConfig, error in
+            XCTAssertNil(oidConfig)
+            XCTAssertEqual(
+                "Error returning discovery document: Test Error Pleasecheck your PList configuration",
```

Comment:
Can you also create an issue to fix this error message? Looks like we're concatenating strings without a space:
https://github.com/okta/okta-sdk-appauth-ios/blob/7fd05261f601a6594761ac03cd77031a6c5ae4ae/Okta/OktaAuth/OktaAuth.swift#L169-L170

Reply:
Done.